### DISCUSSION TITLE
Differences with MetaGraphsNext

### DISCUSSION THREAD

**Discussion by gdalle** [simonschoelly/SimpleValueGraphs.jl#28] [General]
Hi there,
I was browsing JuliaGraphs and wondered why there were two packages for type-stable metagraphs. In what ways is SimpleValueGraphs so different from MetaGraphsNext? Would it make sense to have only one package to turn to?

**Comment by simonschoelly** [comment]
I am not that familiar with MetaGraphsNext - last time I tried to use it I could not get it to work. It still seems to be using dictionaries like MetaGraphs.jl, although it seems to be more type stable. SimpleValueGraphs also defines an abstract interface for graphs with values, which MetaGraphsNext does not.

But there is place for of different graph types in the Julia ecosystem, and as long as they implement the LightGraphs interface they are at least somehow compatible with each other.

**Reply by gdalle** [reply]
OK thanks :) I was just asking since you appear among the contributors of `MetaGraphsNext` as well

**Reply by simonschoelly** [reply]
I think that is because I might have done some minor contribution to MetaGraphs.jl - MetaGraphsNextt.jl was forked from that.
